synapse/lib/types.py

```diff
@@ -716,6 +717,11 @@ def postTypeInit(self):
         if restr is not None:
             self.regex = regex.compile(restr)
 
+        self.envals = None
+        enumstr = self.opts.get('enums')
+        if enumstr is not None:
+            self.envals = enumstr.split(',')
```

Comment:
We might want to strip all of these

Reply:
i don't think we want to be in the business of doing anything to a enum string other than "split". even then, that kind of says `,` is a reserved character.